require-gutenberg/require-gutenberg.php

```diff
@@ -0,0 +1,329 @@
+<?php
+/**
+ * Plugin Name: Require Gutenberg
```

Comment:
```suggestion
 * Require Gutenberg
```

^ Super-minor suggestion, since this the README doesn't suggest including this as a plugin.

Reply:
:laughing: you're right, It was just easier to develop as a plugin 'cause I could test with multiple themes that way.

fixed :+1: